src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VariantMongoWriter.java

```diff
@@ -51,6 +52,7 @@ public VariantMongoWriter(String collection, MongoOperations mongoOperations,
         this.variantToMongoDbObjectConverter = variantToMongoDbObjectConverter;
         this.mongoOperations = mongoOperations;
         this.collection = collection;
+        setTemplate(mongoOperations);
```

Comment:
How did this work previously? Could this cause any conflicts with the use of a specific converter?